src/app/lib/constants/typography.js

```diff
@@ -0,0 +1,14 @@
+import { css } from 'styled-components';
+import mediaQuery from '../../helpers/mediaQueries';
+
+// Font styling below is a subset of BBC GEL Typography "Long Primer"
+const LONG_PRIMER = css`
```

Comment:
Can you export this with a a prefix of `T_` for typography, to match our format for colour constants